avo-hq/avo
# fix: wrong key deleted in key-value

# Description


Fixes https://github.com/avo-hq/avo/issues/682

We also had to update the stimulus to V3 and `stimulus-use` dep.

# Checklist:



## Manual review steps


Description...

1. Go to a project
2. make sure the meta field has more than one row
3. delete the last row
4. expect that the last row was deleted and not the first one

Manual reviewer: please leave a comment with output from the test if that's the case.
